src/MSBuild/LiveLogger/LiveLogger.cs

```diff
@@ -468,6 +473,12 @@ private void MessageRaised(object sender, BuildMessageEventArgs e)
                     _projects.TryGetValue(new ProjectContext(buildEventContext), out Project? project))
                 {
                     ReadOnlyMemory<char> outputPath = e.Message.AsMemory().Slice(index + 4);
+
+                    if (outputPath.Span.Slice(0, _initialWorkingDirectory.Length).SequenceEqual(_initialWorkingDirectory.AsSpan()))
```

Comment:
This comparison should be case insensitive on Windows. The strings we get here from the `CopyFilesToOutputDirectory` message include a user-specified part that could have a different casing than what `Environment.CurrentDirectory` returns.

![image](https://user-images.githubusercontent.com/12206368/233923001-e4f7fe9c-42b0-42ce-92d4-6370202926f8.png)